packages/pyright-internal/src/parser/tokenizer.ts

```diff
@@ -135,6 +135,8 @@ const _byteOrderMarker = 0xfeff;
 
 const _maxStringTokenLength = 32 * 1024;
 
+export const DefaultTabSize = 8;
```

Comment:
For consistency with the rest of the code base, please make constant names lower case.

Reply:
okay